src/formats/kml/KmzFile.js

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Please update to the new file header format.

Reply:
Done